docs/sources/faq.md

```diff
@@ -259,6 +259,10 @@ If you *really* need to change its setting, make sure to change it
 back to its [default value](https://www.debian.org/releases/stable/i386/ch05s03.html.en) 
 afterwards.
 
+### Why do I get `Connection reset by peer` when making a request to a service running in a container?
+
+Typically, this message is returned if the service is already bound to your localhost. As a result, requests coming to the container from outside are dropped. To correct this problem, change the service's configuration on your localhost so that the service accepts requests from all IPs. 
```

Comment:
Yeah,  I'm leaning to pushing as is.  A pseudo example can be worse than none at all for inexperienced people.  Most folks can google like I did --- certainly the text is enough to point them to a good google.  And, more importantly, this better than nothing which is what we have now.

Wrapped to 80 whole file and checked.

If you agree can you LGTM? Maybe you can recommend one other tech reviewer and tag them for me?